src/__tests__/Microcosm-test.js

```diff
@@ -121,14 +159,14 @@ describe('Microcosm', function() {
     })
   })
 
-  describe('Microcosm::deserialize', function() {
-    it ('can handle undefined arguments in deserialize', function() {
-      app.addStore({ toString() { return 'fiz' } })
+  describe('::deserialize', function() {
+    it ('handles undefined arguments', function() {
+      app.addStore('fiz', {})
       app.deserialize()
     })
```

Comment:
so this is just testing that no errors are thrown? Dunno why I never thought to do that.